OpenRA.Mods.Common/Traits/Air/Aircraft.cs

```diff
@@ -746,15 +743,6 @@ public void ResolveOrder(Actor self, Order order)
 				}
 
 				UnReserve();
-
-				// TODO: Implement INotifyBecomingIdle instead
```

Comment:
I know its a pain, but it would reduce the risk of this PR a lot if we could split this out into its own prereq PR.  This is introducing a plumbing change, which is quite different (wrt testing and types of regression) to the activity changes that make up the bulk of this PR.

Reply:
#15873